Application/EdFi.Ods.Api.IntegrationTestHarness/appsettings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "EdFi_Ods": "Server=(local); Database=EdFi_Ods_Populated_Template_Test; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_Admin": "Server=(local); Database=EdFi_Admin_Test; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_Security": "Server=(local); Database=EdFi_Security_Test; Trusted_Connection=True; Persist Security Info=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_master": "Server=(local); Database=master; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;"
+  },
+  "ApiSettings": {
+    "OwinAppStartup": "Startup",
+    "BearerTokenTimeoutMinutes": "30",
+    "ExcludedExtensionSources": "",
+    "SandboxPopulatedTemplateScript": "GrandBend",
+    "SelfHostBaseAddress": "http://localhost:8765/",
```

Comment:
the keys you are adding should not really be in the ApiSettings node, but as separate nodes in the configuration. does this make sense?

Reply:
restructured json file